lib/providers/services/esr_service.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:typed_data';
+
+import 'package:eosdart_ecc/eosdart_ecc.dart';
```

Comment:
This code is pointing to master v2
All new code for v2 should be inside the V2 folder. 

This is a service, should be inside the v2 services folder. `remote`